### PR TITLE
Add version history for 5.0.7

### PR DESCRIPTION
**Pull request by melissalinkert**
As usual, the date probably needs updating once we're closer to tagging.

### CODE REVIEW

File: docs/sphinx/about/whats-new.txt [hunk 1]
```diff
@@ -1,6 +1,15 @@
 Version history
 ===============
 
+5.0.7 (2015 February 5)
+-----------------------
+
+* Several bug fixes, including:
+    - ND filter parsing for Deltavision
```
Comment (mtbc) [inline docs/sphinx/about/whats-new.txt]: capital V ?

**Comment by joshmoore** [comment]
Other PRs that might be worthwhile:

 * Updated/improved documentation ("even more in 5.1"(TM)) - 1449, 1445
 * Minor core formats (OME/XML/TIFF) improvements - 1520, 1475, 1470

But perhaps that's more just for Helen & the announcement.

**Comment by hflynn** [comment]
Docs improvements very minor on this branch really, I'm happy just to mention in passing in the release statement.

**Comment by hflynn** [comment]
Missing capital V aside, this looks good to me.

**Comment by melissalinkert** [comment]
```Deltavision``` has been used exclusively in the version history so far - if I change it for 5.0.7, do you want me to change it for all previous versions too for consistency?

**Comment by hflynn** [comment]
Sorry, I hadn't spotted that! Yes, replace all would be ideal, thanks

**Comment by melissalinkert** [comment]
Commit pushed.

**Comment by hflynn** [comment]
Looks good thanks

**Comment by melissalinkert** [comment]
--rebased-to #1574

**Pull request merged**